src/components/network/selection-creation-panel.tsx

```diff
@@ -226,11 +277,10 @@ const FilterCreationPanel: React.FC<FilterCreationPanelProps> = ({
                         onClick={() => {
                             formMethods.trigger().then((isValid) => {
                                 if (isValid && defaultFolder) {
-                                    onSaveFilter(
-                                        formMethods.getValues() as IFilterCreation,
-                                        defaultFolder
-                                    );
-                                    generateFilterName();
+                                    const data =
+                                        formMethods.getValues() as IFilterSelection;
```

Comment:
It's better to type useForm instead :
const formMethods = useForm<IFilterSelection>(...)

if you do this, "getValues()" will be typed as you expected

Reply:
you can use "SubmitButton" from commons-ui

Reply:
done